source/whats-new.txt

```diff
@@ -12,12 +12,26 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 2.22 <version-2.22>`
 * :ref:`Version 2.21 <version-2.21>`
 * :ref:`Version 2.20 <version-2.20>`
 * :ref:`Version 2.19 <version-2.19>`
 
 .. _upcoming-breaking-changes:
 
+.. _version-2.22:
+
+What's New in 2.22
+------------------
+
+The 2.22 driver release includes the following new features:
+
+- Improvements and bug fixes to the LINQ3 integration.
+- Added support for ``$set`` and ``$vectorSearch`` aggregation stages.
+- Added synonym support for Atlas Search.
+- Reduced memory allocation when using encryption.
+- Added additional logging messages.
```

Comment:
Issue:
The vale action picked up an issue with the term "additional". Please address these or open a new JIRA ticket for the [vale epic](https://jira.mongodb.org/browse/DOCSP-31883) if it is a false positive or if there is some other issue with it.

Suggestion:
I think both "additional" and "more" don't help in understanding the change. I think providing some details or linking to information on what type of logging messages were added would be helpful for users.